src/chain/chain.rs

```diff
@@ -1815,13 +1816,14 @@ mod tests {
         rng::MainRng,
         unwrap,
         xor_space::{Prefix, XorName},
-        BlsSecretKeySet, ConnectionInfo,
+        ConnectionInfo,
     };
     use rand::Rng;
     use std::{
         collections::{BTreeMap, HashMap},
         str::FromStr,
     };
+    use threshold_crypto::SecretKeySet;
```

Comment:
this would make search and replace more difficult, probably best to stick with `bls::SecretKeySet`.